osu.Game/Graphics/UserInterfaceV2/OsuDirectorySelectorHiddenToggle.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Cursor;
+using osu.Framework.Localisation;
+using osu.Game.Graphics.UserInterface;
+using osuTK.Graphics;
+
+namespace osu.Game.Graphics.UserInterfaceV2
+{
+    internal class OsuDirectorySelectorHiddenToggle : OsuCheckbox, IHasTooltip
+    {
+        public LocalisableString TooltipText => @"Show hidden items";
+
+        public OsuDirectorySelectorHiddenToggle()
+        {
+            RelativeSizeAxes = Axes.None;
+            AutoSizeAxes = Axes.Both;
+            Anchor = Anchor.CentreLeft;
+            Origin = Anchor.CentreLeft;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OsuColour colours)
+        {
+            Nub.AccentColour = colours.GreySeaFoamLighter;
+            Nub.GlowingAccentColour = Color4.White;
+            Nub.GlowColour = Color4.White;
+        }
```

Comment:
After updating the file/directory test scenes to use `ThemeComparsionTestScene`, it appears the nub doesn't follow the same colour scheme as the selector:

<img width="726" alt="CleanShot 2022-07-11 at 20 21 43@2x" src="https://user-images.githubusercontent.com/22781491/178321734-10277a90-94c9-447f-b68c-3cfdfa433756.png">

Probably want to resolve `OverlayColourProvider` here and use it, with the `GreySeamFoamLighter` colour as a fallback. Similar to other components like `OsuDirectorySelectorDirectory.Background`.